indexer/src/main/java/uk/ac/ebi/uniprot/indexer/uniprotkb/listener/UniProtKBLogStepListener.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Created by sahmad on 29/01/19 19:28
+ * UniProt Consortium.
+ * Copyright (c) 2002-2019.
+ *
+ */
+
+package uk.ac.ebi.uniprot.indexer.uniprotkb.listener;
```

Comment:
nope, explained when chatting earlier. in brief, need to use a specific one now because we spring batch gets the write + error counts wrong (since we do not use their retry logic).